occurrences/tests/test_api.py

```diff
@@ -686,7 +686,7 @@ def test_delete_study_group_staff_user(staff_api_client, study_group):
 
 
 def test_enrol_not_started_occurrence(snapshot, api_client):
-    # Current date is freezed on 2020-01-04:
+    # Current date is froze on 2020-01-04:
```

Comment:
I don't normally care that match about grammar in code as long as the text is understandable (probably because I suck in it), but now that this was explicitly changed, I have to say I don't think the new version is exactly correct either 🙂 Maybe `is frozen` ?

Reply:
Damn it 🤣  It could have been either `froze` or `is frozen`